xcube_sh/sentinelhub.py

```diff
@@ -39,6 +38,55 @@
 from .version import version
 
 
+class SentinelOAuth2Session(requests_oauthlib.OAuth2Session):
```

Comment:
Please extract all attributes to be serialized in a list and assign it a class readonly attribute `_SERIALIZED_ATTRS`.